assets/src/stories-editor/blocks/amp-story-page/edit.css

```diff
@@ -126,7 +126,7 @@ div[data-type="amp/amp-story-page"] .editor-inner-blocks .editor-block-list__lay
 
 .editor-block-list__layout div[data-type="amp/amp-story-page"][data-amp-selected="parent"] .editor-block-drop-zone {
 	height: 583px;
-	width: 438px;
+	width: 558px;
```

Comment:
Note: this is for extending the dropzone to be able to move blocks over the right edge of a page, too.

Reply:
So this is basically the size of the page plus some padding? Can we use some CSS variables for that to make this more clear?

Reply:
Done in 33a7d6d.